modules/ols-granting-access-to-user-group.adoc

```diff
@@ -23,4 +23,37 @@ This procedure explains how to grant access to a user group. If your cluster has
 ----
 $ oc adm policy add-cluster-role-to-group \
 lightspeed-operator-query-access <group_name>
+----
++
+Alternatively, you can use a YAML file when granting access to a user group by using the following command:
++
+[source,terminal]
+----
+$ oc adm policy add-cluster-role-to-group lightspeed-operator-query-access <group_name> -o yaml --dry-run
+----
++
+The terminal returns the following output:
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  creationTimestamp: null
+  name: lightspeed-operator-query-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: lightspeed-operator-query-access
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: <group_name>
```

Comment:
Does the user need to update the `<group_name>` before creating the object?

Reply:
updated the same way as the other instance.